blocks/library/button/index.js

```diff
@@ -210,7 +198,7 @@ export const settings = {
 		const linkClass = 'wp-block-button__link';
 
 		return (
-			<div className={ `align${ align }` }>
```

Comment:
This is the only unfortunate thing about these "magic" supports. It becomes very non-obvious why we need the `div` wrapper at all.

Reply:
Yes, this might have up to 3 class names appended here :)